### PR TITLE
Use Event SNS.Message

### PR DESCRIPTION
**Pull request by aaron-collier**
Feel free to ignore this @jcoyne, I just wanted to capture the change for now (for documentation)

### CODE REVIEW

File: lambda/README.md [hunk 1]
```diff
@@ -8,18 +8,17 @@ zip lambda.zip main
...
 1. Upload zip and create a function definition
 ```
-AWS_ACCESS_KEY_ID=999999 AWS_SECRET_ACCESS_KEY=1231 aws \
+AWS_REGION=localstack AWS_ACCESS_KEY_ID=999999 AWS_SECRET_ACCESS_KEY=1231 aws \
 --endpoint-url http://localhost:4574 lambda create-function \
 --function-name f1 \
 --runtime go1.x \
 --role r1 \
 --handler main \
---environment "Variables={ES_HOST=192.168.2.7}" \
```
Comment (jcoyne) [inline lambda/README.md]: Why is this removed?

File: lambda/README.md [hunk 2]
```diff
@@ -33,20 +32,32 @@ AWS_ACCESS_KEY_ID=999999 AWS_SECRET_ACCESS_KEY=1231 aws es \
...
-AWS_ACCESS_KEY_ID=999999 AWS_SECRET_ACCESS_KEY=1231 aws lambda \
---endpoint-url=http://localhost:4574 invoke \
---function-name f1 result.log \
---payload '{"Records": [{"EventSource": "foo", "Sns": { "Timestamp": "2014-05-16T08:28:06.801064-04:00", "Message": "Hello world!" }}]}'
+awslocal sns create-topic --name rialto-msg
+```
+
+1. Subscribe the lamba function to the SNS Topic
+```
+awslocal sns subscribe \
```
Comment (jcoyne) [inline lambda/README.md]: Why `awsloca`l here and `aws` on line 16?

File: lambda/README.md [hunk 3]
```diff
@@ -33,20 +32,32 @@ AWS_ACCESS_KEY_ID=999999 AWS_SECRET_ACCESS_KEY=1231 aws es \
...
+  --topic-arn arn:aws:sns:us-east-1:123456789012:rialto-msg \
+  --protocol lambda \
+  --notification-endpoint arn:aws:lambda:localstack:000000000000:function:f1
+```
+
+1. Publish to the SNS Topic
+```
+awslocal sns publish \
+   --topic-arn arn:aws:sns:us-east-1:123456789012:test-topic \
```
Comment (jcoyne) [inline lambda/README.md]: Why are you sending a message to a topic you haven't subscribed to?

File: lambda/README.md (hunk 1 above)
Comment (aaron-collier) [reply lambda/README.md]: I think I'm behind when you added that. Will get it back in, thanks for the heads up.

File: lambda/README.md (hunk 2 above)
Comment (aaron-collier) [reply lambda/README.md]: I wasn't able to get aws to run properly (will have to check the error) and only got this to run successfully with `awslocal`

File: lambda/README.md (hunk 3 above)
Comment (aaron-collier) [reply lambda/README.md]: copy-pasta, fixing.

File: lambda/README.md (hunk 2 above)
Comment (jcoyne) [reply lambda/README.md]: awslocal is actually just wrapping aws: https://github.com/localstack/awscli-local/blob/master/bin/awslocal#L89
